module/extension/FileWatcher/src/FileWatcher.cpp

```diff
@@ -90,6 +90,8 @@ namespace extension {
                 }
             }
         }
+        // Shut compiler up about unused argument by pretending to use it
+        (void) status;
```

Comment:
Rather than having an unused argument used like this, you should either be removing the variable or using it for something. So, in this case, change the function signature to
```cpp
void FileWatcher::file_watch_callback(uv_fs_event_t* handle, const char* filename, int events, int /*status*/) {
```

Reply:
I tried that, but it says that the function is used by /usr/local/include/uv.h:1586:48: when I try to build it, so it looks to me like the function is referenced by another library, and requires that function signature

Reply:
Don't remove the last argument, just the variable name that it's going in. So leave the int part only

Reply:
Roger that